packages/messaging/README.md

```diff
@@ -0,0 +1,20 @@
+
+# React Native AEPMessaging Extension
+
+[![npm version](https://badge.fury.io/js/%40adobe%2Freact-native-aepmessaging.svg)](https://www.npmjs.com/package/@adobe/react-native-aepmessaging) 
+[![npm downloads](https://img.shields.io/npm/dm/@adobe/react-native-aepmessaging)](https://www.npmjs.com/package/@adobe/react-native-aepmessaging)
+
+## Usage
+
+### [Messaging](https://github.com/adobe/aepsdk-messaging-ios#adobe-experience-platform---messaging-extension-for-ios)
```

Comment:
Doc should go to here? https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-journey-optimizer